internal/store/bigtable/group.go

```diff
@@ -69,37 +69,50 @@ func NewGroup(
 	}
 }
 
-// GetFrequentGroup creates a group that only has frequent import group table.
-func GetFrequentGroup(g *Group) *Group {
+// GetHighestRankGroup creates a group that only has the highest rank import
+// group table.
+func GetHighestRankGroup(g *Group) *Group {
+	SortTables(g.tables)
 	result := &Group{tables: []*Table{}}
 	for _, t := range g.tables {
-		if strings.HasPrefix(t.name, "frequent_") {
-			result.tables = append(result.tables, t)
-			break
-		}
+		result.tables = append(result.tables, t)
+		break
 	}
 	return result
 }
 
 // SortTables sorts the bigtable by import group preferences
-//   - frequent should always be the highest rank
-//   - infrequent should always be the lowest rank
-//   - if a group is not in ranking list, put it right before "infrequent" and
+//   - custom import group shold be the highest rank
+//   - For non-custom import groups:
```

Comment:
For my understanding, branch is handled completely differently?

Reply:
Missed in here, branch cache ranks highest in non-custom import groups.